public/index.php

```diff
@@ -3,62 +3,30 @@
 /**
  * This file is part of the Vökuró.
  *
- * (c) Phalcon Team <team@phalconphp.com>
+ * (c) Phalcon Team <team@phalcon.io>
  *
  * For the full copyright and license information, please view
  * the LICENSE file that was distributed with this source code.
  */
 
-use Phalcon\DI\FactoryDefault;
-use Phalcon\Mvc\Application;
+use Vokuro\Application as VokuroApplication;
 
 error_reporting(E_ALL);
```

Comment:
Do we want to do full error reporting by default?

Reply:
I would leave it for now, as there were no errors handler and I didn't want to do it on current iteration.